bundle/app/manifests/sonataflow-operator.clusterserviceversion.yaml

```diff
@@ -528,13 +599,9 @@ spec:
   - serverless
   - serverlessworkflow
   links:
-  - name: Product Page
-    url: https://sonataflow.org/serverlessworkflow/latest/index.html
-  maintainers:
-  - email: bsig-cloud@redhat.com
-    name: Red Hat
+  - name: SonataFlow Operator
+    url: https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/index.html
   maturity: alpha
-  minKubeVersion: 1.23.0
   provider:
-    name: Red Hat
```

Comment:
isn't is supposed to be Red Hat?

Reply:
Probably for the community version it won't be Red Hat but Apache.
@ricardozanini can you confirm?

Reply:
Provider should be Red Hat, we change to Apache once we do the move.

Reply:
I reverted it to Red Hat